### PR TITLE
Add responsive nav bar

### LINKED ISSUE #11: Add responsive nav bar to website
Nav link names and associated URLs:

- What is Pursuance? `#what`
- Media `#media`
- Team `#team`
- FAQ `#faq`
- Contact `#contact`
- Donate `#donate`
- Broader Vision `#vision`

### PR DESCRIPTION
**Pull request by seabassgonzalez**
Fixes #11

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,231 +6,254 @@
...
+        <i class="fa fa-bars menu"></i>
+      </div>
+      <ul>
+        <li><a href="#what">What is Pursuance?</li>
+        <li><a href="#">Media</li>
+        <li><a href="#">Team</li>
+        <li><a href="#">FAQ</li>
+        <li><a href="#">Contact</li>
+        <li><a href="#">Donate</li>
+        <li><a href="#">Broader Vision</li>
```
Comment (elimisteve) [inline index.html]: @seabassgonzalez Add closing `</a>` tags :-)

File: index.html [hunk 2]
```diff
@@ -6,231 +6,254 @@
...
+    </p>
+
+    <a class="btn btn-primary nav-link" href="/donate.html">Donate to Pursuance!</a>
+    <br />
+    <br />
+  </div>
```
Comment (elimisteve) [inline index.html]: @seabassgonzalez Please don't change the indentation level if we're not also editing the text; someone else will soon create a PR where the "Broader Vision" section is changed and avoiding merge conflicts based on mere whitespace differences would be great!

Comment (seabassgonzalez) [reply index.html]: Got it

**Comment by elimisteve** [comment]
@seabassgonzalez Please remove the ~3 `.DS_Store` files

**Comment by seabassgonzalez** [comment]
Removed

**Comment by seabassgonzalez** [comment]
is there an existing subsection that should be used as the FAQ?

**Comment by seabassgonzalez** [comment]
Changed padding such that the clickable area encompasses the whole li region

**Pull request merged**